docs/multi-stage-query/api.md

```diff
@@ -144,7 +144,7 @@ GET /druid/indexer/v1/task/<taskId>
 
 ```bash
 # Make sure you replace `username`, `password`, `your-instance`, `port`, and `taskId` with the values for your deployment.
-curl --location --request GET 'https://<username>:<password>@<hostname>:<port>/druid/indexer/v1/task/<taskId>/status'
+curl --location --request GET 'https://<username>:<password>@<your-instance>:<port>/druid/indexer/v1/task/<taskId>/status'
```

Comment:
can we please change this to pass credentials in the header rather than as part of the URL. See: https://github.com/apache/druid/issues/13193. Looks like this was done in line 105.

For curl it would use this syntax:
`curl --user username:password GET 'https://<your-instance>:<port>/druid/indexer/v1/task/<taskId>/status'`